### PR TITLE
added js1/week2/homework

### PR DESCRIPTION
**Pull request by rajanibusani**
js1/week2/homework

### CODE REVIEW

File: javascript/javascript1/week2/index.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+const fullname2= getFullname('Mark', 'wiens',false);
+console.log(fullname1);
+console.log(fullname2);
+
+//Event day
+let day = new Date();
+let today = day.getDay();
+const sevenDays = ['Sunday','Monday', 'Tuesday','Wednesday','Thursday','Friday','Saturday'];
+function getEventWeekday(eventInDays){
+    let eventDay = (today+eventInDays)%7;
```
Comment (aina21) [inline javascript/javascript1/week2/index.js:27]: you can use const

File: javascript/javascript1/week2/index.js [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+const class16Students = [];
+function addStudentToClass(studentName) {  
+if(studentName === "") {
+    console.log('You cannot add an empty string to a class');
+}else if (class16Students.includes(studentName)){
+    console.log(`Student ${studentName} is already in the class.The same person cannot be added to the class.`)
+}else if (class16Students.length<6){
+  class16Students.push(studentName);}
+else if(class16Students.length=6){
```
Comment (aina21) [inline javascript/javascript1/week2/index.js:59]: Do you mean "class16Students.length===6"

File: javascript/javascript1/week2/index.js [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+    if(studentName === "Queen Margrethe II" || class16Students.includes('Queen Margrethe II')){
```
Comment (aina21) [inline javascript/javascript1/week2/index.js:60]: what happened if we add "Queen Margrethe II" more than one time?

**Review by aina21 — APPROVED** [review]
Good job Rajani 👍

File: javascript/javascript1/week2/index.js (hunk 2 above)
Comment (aina21) [inline javascript/javascript1/week2/index.js:59]: Do you mean class16Students.length>=6

**Review by aina21 — CHANGES_REQUESTED** [review]

File: javascript/javascript1/week2/index.js (hunk 1 above)
Comment (rajanibusani) [reply javascript/javascript1/week2/index.js:27]: Hi Ida, yes, i will change to const...

File: javascript/javascript1/week2/index.js (hunk 3 above)
Comment (rajanibusani) [reply javascript/javascript1/week2/index.js:60]: its going to logout "Queen is already in the class...! as  i gave the condition above...if .includes.(studentName);

File: javascript/javascript1/week2/index.js (hunk 2 above)
Comment (rajanibusani) [reply javascript/javascript1/week2/index.js:59]: sorry Ida, i meant class16students==6 here, plz correct me if am wrong, what my thought is if  .length==6 means, if 7th student is  queen or if queen is already there, me allowing that student..! and after that logging out no more students....is it right or am i missing some logic? Tq

**Pull request merged**